shared/chat/conversation/messages/react-button/emoji-picker/index.tsx

```diff
@@ -35,12 +37,13 @@ const getData = memoize((topReacjis: Array<string>) => {
 
   // Get emoji results for a query and map
   // to full emoji data
-  const getFilterResults = filter =>
+  const getFilterResults = (filter: string): Array<Data.EmojiData> =>
     emojiIndex
+      // @ts-ignore type wrong?
       .search(filter, {maxResults: maxEmojiSearchResults})
-      .map(res => emojiNameMap[res.id])
+      .map((res: {id: string}) => emojiNameMap[res.id])
       // MUST sort this so its stable
-      .sort((a, b) => a.sort_order - b.sort_order)
+      .sort((a: any, b: any) => a.sort_order - b.sort_order)
```

Comment:
Can we not narrow these types down further? This seems like it could lead to some `property sort_order does not exist` errors later on

Reply:
we have to fix the types upstream